mozilla_version/test/test_gecko.py

```diff
@@ -742,3 +742,1141 @@ def test_gecko_version_hashable():
     It is possible to hash `GeckoVersion`.
     """
     hash(GeckoVersion.parse('63.0'))
+
+
+@pytest.mark.parametrize('version_string', (
+    "1.0",
+    "1.5",
+    "2.0",
+    "3.0",
+    "3.5",
+    "3.6",
+    "4.0",
+    "13.0",
+    "14.0.1",
+    "14.0.1-build1",
+    "15.0",
+    "33.0",
+    "33.1",
+    "33.1-build1",
+    "33.1-build2",
+    "33.1-build3",
+    "34.0",
+    "124.0",
+    "125.0.1",
+    "125.0.1-build1",
+))
+def test_firefox_version_are_major(version_string):
```

Comment:
Note to future readers: we don't have tests to check whether a version is not a major one. That said, it's already covered by the error reporting GeckoVersion.__attrs_post_init__(). Thus, a version wrongly considered as a major one will raise an error and make other tests fail.